server/routes/events.js

```diff
@@ -132,6 +132,7 @@ router.post("/:EVENT_COLLECTION", (req, res) => Project.findOne({ projectId: req
     }
     cenote.id = uuid();
     payload[i].cenote = { ...cenote };
+    if (project.redisHist) payload[i].redisHist = true;
```

Comment:
`redisHist` is an implementation detail, it's not part of the payload. As such, I think you should put it inside the `cenote` object.

Reply:
Ok, the historical averages functionality will be put under the microscope in its entirety, so I'll take care of it.

Reply:
Done.